genon/serving/paddle/etc/smoke_test_inference.py

```
@@ -0,0 +1,108 @@
+        help="OCR 대상 이미지 경로. 미지정 시 임시 합성 이미지 사용",
+    )
+    args = parser.parse_args()
+
+    print(f"[smoke] paddle device check ...", flush=True)
+    device = _assert_gpu()
+    print(f"[smoke] paddle device = {device}", flush=True)
+
+    if args.image:
+        image_path = Path(args.image)
+    else:
+        image_path = Path("/tmp/ocr_smoke_sample.png")
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/ocr_smoke_sample.png"

(S108)